src/ManageCourses.ApiClient/GitVersion.txt

```diff
@@ -1 +1 @@
-0.8.0
\ No newline at end of file
+0.8.1
```

Comment:
master is at 0.9.0
this needs updating

reject

Reply:
Good catch!! Fixed :+1: